soap-attachments/3.0/_index.md

```diff
@@ -1,21 +1,36 @@
 ---
-title: "Jakarta SOAP with Attachments 3.0 (under development)"
+title: "Jakarta SOAP with Attachments 3.0"
 date: 2021-04-15
```

Comment:
Shouldn't this be closer to today? (I know it can't be in the past, but April, last year doesn't make sense.)

Reply:
it actually makes sense, see https://github.com/jakartaee/specifications/pull/427#issuecomment-996523744 for the reason